### ISSUE TITLE
Offsets not restoring after SteamVR closes

### ISSUE THREAD

**Issue by Piemur1** [matzman666/OpenVR-InputEmulator#150]
I can't seem to save the offset settings I input for my devices. When I start up SteamVR I have to go into the settings every time and fix them again. Is there a configuration file I can go to that will store these settings for me? If I attempt to save it into a profile, then when I load the profile, my view goes very weird, with like a window only directly above and below, but nothing shows anywhere else. This only really happens when I turn on the tracker. Leaving the HMD and Controllers only, it works fine, and I don't even need to calibrate anything, but once the tracker and the lighthouses for HTC kick on, everything gets wonky.

HMD: Oculus Rift
Controllers: Oculus Touch
Hip Tracker: HTC Vive Tracker 2018
HTC Tracker Peripherals: HTC Base Stations not connected to PC

**Comment by cr4zyw3ld3r** [comment]
Agreed, if there is an option to force auto load of the offsets on start that would be amazing. But if not can we at least know if there is a way to make steamvr load the controller settings?
